.github/workflows/test-environment.yml

```
@@ -81,11 +81,14 @@ jobs:
             exit 1
           fi
 
-      - name: Mask API Key
+      - name: Mask Keys
```

If you are looking for a more general but informative option.

```suggestion
      - name: Mask Sensitive Data
```